### DISCUSSION TITLE
How to filter out by criteria, e.g. not having this label, or not show low severity/priority issues?

### DISCUSSION THREAD

**Discussion by BulatKha** [community/community#38063] [Projects and Issues] [answered]
Can't really understand how to filter out issues via excluding by some criteria, e.g. status/type/priority/size. For instance I do not want to see the low priority tasks in the view, or just the tasks for QA, therefore not having some labels like "Dev-QA" or something similar.

**Comment by ossanna16** [comment]
Hi there @BulatKha and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by vvdavidova** [comment, marked as answer]
Hey @BulatKha, you can try by adding a - (dash) before your status, so -status:typeinwhateveryouwanttoexcludehere or -label:yourexclusionhere.  You can also combine a few of those exclusions. 
I hope that helps. :)
